gnovm/pkg/gnolang/machine.go

```diff
@@ -1820,20 +1855,40 @@ func (m *Machine) LastCallFrame(n int) *Frame {
 			}
 		}
 	}
-	panic("frame not found")
+
+	if !safe {
+		panic("frame not found")
+	}
```

Comment:
The way you use `safe` is disturbing, but maybe it's just me. 

When I see a *safe* method, I expect to panic if the expected conditions are not met. Here it's the opposite, if it's *safe*, you don't panic...

Reply:
I guess it depends on perspective. I'm open to a different name

Reply:
My take on this:
- `LastCallFrame(n)` panics when there's no frame at `n` (no change)
- `UnsafeLastCallFrame(n)` or `LastCallFrameUnsafe(n)` doesn't panic.

I'd like to get others' opinions on what is a *safe* behavior, because for me it's when it panics.

An other option:
- rename `LastCallFrame(n)` to `MustLastCallFrame(n)`
- rename `LastCallFrameSafe(n)` to `LastCallFrame(n)`
The `Must` prefix is well known to indicate that a panic can happen. We remove that *safe* usage that might be ambiguous.

Reply:
Yeah I like that idea